crates/actors/src/mempool_service/lifecycle.rs

```diff
@@ -85,7 +88,362 @@ impl Inner {
         // 6. Similar work with commitment transactions (stake and pledge)
         //    - This may require adding some features to the commitment_snapshot so that stake/pledge tx can be rolled back and new ones applied
 
-        tracing::info!("Reorg handled, new tip: {}", event.new_tip.0.to_base58());
+        // needs to be done:
+        // re-org support for migrated blocks
```

Comment:
Is this still a todo item? If so, is it worth marking as a todo?

```suggestion
        // TODO: re-org support for migrated blocks
```